build.sh

```diff
@@ -114,6 +115,9 @@ function usage()
     echo "                                 parameter is not present, cmake executable     "
     echo "                                 should be available from your \$PATH variable. "
     echo "                                 will clean all the generated code and binaries "
+    echo "    --add-cmake-args <s>         Additional defines and arguments that need to  "
```

Comment:
I think this would be more clear if it said "Additional defines and arguments that *will be* passed to the cmake executable..." Since on many platforms you will not need these arguments

Reply:
Done